src/main/java/io/deepstream/rpc/RpcHandler.java

```diff
@@ -55,11 +55,11 @@ public void unprovide( String name ) {
     }
 
     public void make(String name, JsonObject data, RpcResponseCallback callback ) {
-        String uid = this.client.getUid();
-        String typedData = MessageBuilder.typed( data );
+        _make( name, MessageBuilder.typed( data ), callback );

```

Comment:
This doesn't need to _make since java does method overloading based on arguments